src/api/journalEntries/journalEntry.ts

```diff
@@ -31,10 +31,11 @@ export const useJournalEntry = (id: string) => {
 };
 
 const createJournalEntryMutation = gql`
-  mutation createJournalEntry($data: JournalEntryCreateInput!){
+  mutation createJournalEntry($data: JournalEntryCreateInputData!){
```

Comment:
This is due to the backend now using the custom resolver instead of the generated one